### ISSUE TITLE
Resizing window result in additionnal blank sidebar

### ISSUE THREAD

**Issue by matthiasbe** [nextcloud/calendar#2276]
### Steps to reproduce
1. Open your calendar
2. Resize window so that the sidebar is hidden and accessible from a button
3. Open the sidebar using this button
4. Resize window to original size

### Expected behaviour
The view after step 4 should be the same as in step 1. 

### Actual behaviour
There is an additionnal "shadow" sidebar showing next to the actual one.
![image](https://user-images.githubusercontent.com/9254232/82297902-9f5ba300-99b3-11ea-8e0b-2995817f4ad8.png)

### Calendar app
**Calendar app version:** (see apps admin page, e.g. 2.0.1)
2.0.3
**CalDAV-clients used:** (Thunderbird Lightning, DAVx5, Evolution, macOS Calendar, etc)

### Client configuration
**Browser:** (e.g. Firefox 48)
Mozilla Firefox 68.7.0esr (64-bit)
**Operating system:** (e.g. Arch Linux)
Debian KDE Plasma
### Server configuration

**Operating system**: (e.g. Debian 8)
Raspbian stretch
**Web server:** (e.g. Apache, Nginx,...)
Apache 2.4.25-3+deb9u9
**Database:** (e.g. MariaDB, SQLite or PostgreSQL)
MariaDB 5.5.9999+default
**PHP version:** (e.g. 7.0.3)
7.3.18-1+0~20200515.59+debian9~1.gbp12fa4f
**Nextcloud Version:** (see admin page, e.g. 17.0.2)
18.0.4
**Updated from an older installed version or fresh install:**
17.0.5, 17.0.6
**List of activated apps:**
```
Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - bruteforcesettings: 1.6.0
  - calendar: 2.0.3
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - dav: 1.14.0
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_external: 1.9.0
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - maps: 0.1.6
  - news: 14.1.7
  - nextcloud_announcements: 1.7.0
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - password_policy: 1.8.0
  - photos: 1.0.0
  - previewgenerator: 2.3.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - recommendations: 0.6.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - support: 1.1.0
  - survey_client: 1.6.0
  - systemtags: 1.8.0
  - text: 2.0.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - viewer: 1.2.0
  - workflowengine: 2.0.0
Disabled:
  - admin_audit
  - encryption
  - user_ldap
```

**Nextcloud configuration:**

```{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.*",
            "*"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "18.0.4.2",
        "overwrite.cli.url": "http:\/\/192.168.0.16\/nextcloud",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "updater.secret": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "mysql.utf8mb4": true,
        "updater.release.channel": "stable",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "logfile": "\/var\/log\/nextcloud.log"
    }
}

```

### Logs
Are logs relevant for this issue ?

**Comment by georgehrke** [comment]
This was most likely fixed with https://github.com/nextcloud/nextcloud-vue/pull/989 already, need to check

**Comment by georgehrke** [comment]
> Are logs relevant for this issue ?

No need for logs here :)

**Comment by georgehrke** [comment]
I was able to reproduce this issue on 2.0.3, but no longer on master.

So this was indeed fixed with nextcloud/nextcloud-vue#989 and was merged into Calendar in https://github.com/nextcloud/calendar/pull/2227

**Comment by georgehrke** [comment]
A fix for this bug will be shipped with 2.1.0
